content/general/functions.textile

```diff
@@ -26,9 +26,9 @@ h2. Payload types
 
 Ably currently supports two types of data that can be delivered via our Functions:
 * "Messages":#messages - messages trigger function calls as soon as they are published on a channel
-* "Presence events":#presence - when clients enter, update their data, or leave channels, the presence event triggers a function call
+* "Presence events":#presence - presence events trigger function calls when clients enter, update their data, or leave channels
 
-You can "configure Functions":#configure from the "Reactor tab in your app":https://support.ably.io/solution/articles/3000074406-how-to-reach-the-reactor-page on a per app basis which can apply to one or more channels in that app. Reactor Functions can be filtered by channel naming using a regular expression, for example @^click_.*_mouse$@.
+You can "configure 'functions'":#configure from the "Reactor tab in your app":https://support.ably.io/solution/articles/3000074406-how-to-reach-the-reactor-page on a per app basis which can apply to one or more channels in that app. Reactor Functions can be filtered by channel naming using a regular expression, for example @^click_.*_mouse$@.
```

Comment:
This is good, mind showing what would match on and what it would not match on in these docs?